src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+    public static final String DIVIDER = "____________________________________________________________";
```

Comment:
For my dividers, I used the `repeat` method of a String, which I thought would make it look cleaner. For example:
```java
"_".repeat(5) // Will become _____
```

My apologies for assuming you didn't know about this method. If you did, could you share why you chose not to do it that way?

Reply:
I actually like that you did not use the String.repeat() function as i think it improves readability of the code. However definitely understand that it is a QOL improvement to use the repeat() function and i also did not know about the repeat function before!